controller/controller.go

```diff
@@ -167,23 +166,11 @@ func (c *Controller) DeleteFunc(obj interface{}) {
 
 	resourceSet, err := c.resourceSet(obj)
 	if IsNoResourceSet(err) {
-		// In case the resource router is not able to find any resource set to
-		// handle the reconciled runtime object, we stop here. Note that we just
-		// remove the finalizer regardless because at this point there will never be
-		// a chance to remove it otherwhise because nobody wanted to handle this
-		// runtime object anyway.
-
-		c.logger.Log("level", "debug", "message", "removing finalizer from runtime object")
-
-		err = c.removeFinalizer(obj)
-		if err != nil {
-			c.logger.Log("level", "error", "message", "stop reconciliation due to error", "stack", fmt.Sprintf("%#v", err))
-			return
-		}
-
-		c.logger.Log("level", "debug", "message", "removed finalizer from runtime object")
-
+		// In case there is no resource set for the object. We should
+		// make sure are resources are deleted for the object and then
```

Comment:
`sure are resources are deleted` -> `sure the resources are deleted` ?

Reply:
This comment no longer exist.